### ISSUE TITLE
VMWare Horizon Client Download recipe failing (Curl)

### ISSUE THREAD

**Issue by jshirle2** [autopkg/scriptingosx-recipes#76]
I am seeing the following error when trying to run the VMWareHorizon download recipe (com.github.scriptingosx.download.VMwareHorizonClient):

```
Traceback (most recent call last):
  File "/Library/AutoPkg/autopkglib/URLGetter.py", line 177, in execute_curl
    errors=errors,
  File "/Library/AutoPkg/Python3/Python.framework/Versions/Current/lib/python3.7/subprocess.py", line 512, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/usr/bin/curl', '--silent', '--show-error', '--no-buffer', '--dump-header', '-', '--speed-time', '30', '--location', '--url', 'https://download3.vmware.com/software/view/viewclients/CART23FQ1/VMware-Horizon-Client-2111.1-8.4.1-19484634.dmg', '--fail', '--output', '/Users/ussadmin/Library/AutoPkg/Cache/com.github.scriptingosx.download.VMwareHorizonClient/downloads/tmpp_rbgu3m']' returned non-zero exit status 22.
```

Looks like the curl command is failing on that URL. 

Thanks!

**Comment by dev-yeet** [comment]
download link format looks like this now: 

https://download3.vmware.com/software/CART23FQ1_MAC_2111_1/VMware-Horizon-Client-2111.1-8.4.1-19484634.dmg

as opposed to: 

https://download3.vmware.com/software/view/viewclients/CART23FQ1/VMware-Horizon-Client-2111.1-8.4.1-19484634.dmg

**Comment by scriptingosx** [comment]
closed with #77
